server/src/main.rs

```diff
@@ -42,6 +43,15 @@ impl Backend {
     }
 }
 
+fn get_dynamo_timestamp(time: SystemTime) -> AttributeValue {
```

Comment:
```suggestion
fn to_dynamo_timestamp(time: SystemTime) -> AttributeValue {
```

Reply:
Good Suggestion. I updated the signature.